docs/contribute/01_environment.qmd

```diff
@@ -67,37 +74,46 @@ for manager, params in managers.items():
     print("    gh repo fork --clone --remote ibis-project/ibis")
     print("    ```")
     print()
-    print(f"1. Create a Conda environment using `environment.yml`")
+    print(f"1. Create a {name} environment using `environment.yml`")
     print()
     print("    ```sh")
     print("    cd ibis")
-    print(f"    {manager} env create --file conda/environment.yml")
+    if manager == "conda" or manager == "mamba":
+        print(f"    {manager} env create --file conda/environment.yml")
+    elif manager == "pixi":
+        print(f"    {manager} init --import conda/environment.yml")
     print("    ```")
     print()
-    if manager == "conda" or manager == "mamba":
-        print()
-        print(
-            "    If you're using arm64 architecture (Mac M1/M2), use "
-            "`conda/environment-arm64.yml` for setting up a dev environment for all the "
-            "backends that are possible to install excluding Flink; use "
-            "`conda/environment-arm64-flink.yml` for setting up a dev environment for all "
-            "the backends that are possible to install including Flink. The reason to have "
-            "two separate environments is because `apache-flink` forces `pyarrow` to "
-            "downgrade to 11.0, which causes conflicts in other backends."
-        )
+    if manager == "pixi":
+        print("    This command will initialize a `pixi.toml` file and also modify ")
+        print("the `.gitignore` to prevent Pixi configuration from being added to ")
+        print("`git`.")
         print()
+    print(
+        "    If you're using arm64 architecture (Mac M1/M2), use "
+        "`conda/environment-arm64.yml` for setting up a dev environment for all the "
+        "backends that are possible to install excluding Flink; use "
+        "`conda/environment-arm64-flink.yml` for setting up a dev environment for all "
+        "the backends that are possible to install including Flink. The reason to have "
+        "two separate environments is because `apache-flink` forces `pyarrow` to "
+        "downgrade to 11.0, which causes conflicts in other backends."
+    )
+    print()
     print(f"1. Activate the environment")
     print()
     print("    ```sh")
-    print(f"    {manager} activate ibis-dev")
+    if manager == "conda" or manager == "mamba":
+        print(f"    {manager} activate ibis-dev")
+    elif manager == "pixi":
+        print(f"    {manager} shell")
     print("    ```")
     print()
     print(
         f"1. Install your local copy of `ibis` into the {manager.capitalize()} environment"
     )
     print()
     print("    ```sh")
-    print("    pip install -e .")
+    print("    uv pip install -e .")
```

Comment:
Where'd `uv` come from?  Should this just be `pip` still?

Reply:
1. `pip` isn't installed by default in the environment created this way (I could, but...)
2. Since #10424, `uv` is included in `conda/environment.yml` anyway, so why not use `uv` over `pip`? Seems there was a decision to generally move to `uv`, and it's what's being used in CI.

Reply:
Ah, I didn't realize that `uv` had also been added to the yml files. All good, then!